Clinical-Genomics/scout
# Show Swedac logo in portrait-oriented delivery report

- Fix missing text from the delivery report, while keeping in in portrait orientation (fix #3126)
- Fix Swedac logo missing from delivery report (fix #3115)

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh your.email@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Install this branch on cg-vm1
2. Pick a case with delivery report, for instance this one: https://scout-dev.scilifelab.se/cust002/F0042222
3. Make sure that the exported PDF delivery report using this branch shows the swedac logo and the page looks fine


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
